pkg/controller/configmap/configmap_controller.go

```diff
@@ -320,7 +324,8 @@ func (r *ReconcileConfigMap) Reconcile(request reconcile.Request) (reconcile.Res
 	}
 
 	// Render configurations
-	objs, err := Render(instance, &ncpReplicas, opNsxSecret, opLbSecret)
+	objs, err := Render(instance, &ncpReplicas, &ncpTolerations, &nsxNodeAgentDsTolerations,
```

Comment:
out of curiosity, why put new parameters in the middle instead of in the end?

Reply:
I just want to goup the parameters for the same resource together,  So, eventually, it calling looks like:
objs, err := Render(instance, &ncpReplicas, &ncpNodeSelector, &ncpTolerations, &nsxNodeAgentDsTolerations,
		opNsxSecret, opLbSecret)

&ncpReplicas, &ncpNodeSelector, &ncpTolerations are for NCP deployment.
nsxNodeAgentDsTolerations for node agent DaemonSet.
the last two is for secret. 

Also, the order of arguments passed in is kept the same one of handling logic inside function.